src/libraries/Common/src/Interop/Windows/BCrypt/Cng.cs

```diff
@@ -243,11 +241,11 @@ protected sealed override bool ReleaseHandle()
 
     internal sealed class SafeKeyHandle : SafeBCryptHandle
     {
-        private SafeAlgorithmHandle _parentHandle = null;
+        private SafeAlgorithmHandle? _parentHandle = null;
 
         public void SetParentHandle(SafeAlgorithmHandle parentHandle)
         {
-            Debug.Assert(_parentHandle == null);
+            Debug.Assert(_parentHandle is null);
```

Comment:
As SafeHandle doesn't define operator== there's no difference between `is null` and `== null` on this line.

Please don't make style changes in this PR.